File: uri/salario/salario.hs
import Text.Printf
import System.IO
import Prelude

fafoo :: Integer -> Double -> Double
fafoo a b = do
        let x = fromInteger a :: Double
        let y = b :: Double
        x * y

main = do
    a <- readLn :: IO Integer
    
    b <- readLn :: IO Integer
    c <- readLn :: IO Double

    let foo = fafoo b c :: Double
    printf "NUMBER = %d\n" a
    printf "SALARY = U$ %.2f\n" foo

File: uri/salario/salario.cs
using System;
using System.Collections.Generic;
using System.Globalization;
using System.Text;


namespace URI
{
    public struct funcsalario
    {
        public Int32 num;
        public Int32 horastrabalhadas;
        public Double porhora;
    }   

    class URI
    {
        static double calculasalario( int horas, double valsalario )
        {
            return (horas * valsalario);
        }
        static void Main( string[] args )
        {
            double result;
            funcsalario func;
            func.num = Convert.ToInt32(Console.ReadLine());
            func.horastrabalhadas = Convert.ToInt32(Console.ReadLine());
            func.porhora = Convert.ToDouble(Console.ReadLine(), CultureInfo.InvariantCulture);
            result = calculasalario(func.horastrabalhadas, func.porhora);
            Console.WriteLine("NUMBER = {0}", func.num);
            Console.WriteLine( String.Format("SALARY = U$ {0:0.00}", result ));
        }
    }
}
